spec/models/institution_builder_spec.rb

```diff
@@ -99,6 +99,15 @@
       end
     end
 
+    describe 'when preventing duplicates' do
+      it 'duplicates in WEAMs are not present in institutions' do
+        create :weam, facility_code: '18181818', institution: 'REAL SCHOOL'
+        create :weam, facility_code: '18181818', institution: 'FAKE SCHOOL'
+        described_class.run(user)
+        expect(institutions.where(facility_code: '18181818').count).to eq(1)
```

Comment:
There may be a more framework specific solution to this here, because I agree it does seem less than ideal to add then delete to prevent the duplicates. Since this was what we were using to resolve this potential duplication issue previously, I kept this basically the same since there were no real issues with it.

Reply:
Also for the specs, we can inspect `FAKE SCHOOL` to be deleted since it has the larger of the two ids since it was added after `REAL SCHOOL`.

Reply:
As a bit of a follow up on why we do it this way, we have no control over the data received and occasionally receive a WEAMs file with duplicate facility codes. Adding a unique constraint here would cause a crash when that bad data does come in, even though that doesn't happen often.